docs/configuration.md

```diff
@@ -150,6 +152,7 @@ of the most common options to set are:
   <td>
     Amount of memory to use for the driver process, i.e. where SparkContext is initialized.
     (e.g. <code>1g</code>, <code>2g</code>).
+    Default unit: MiB
```

Comment:
Everywhere the default isn't bytes, a clause like ", in MiB unless otherwise specified", seems cleanest. There are 9 such properties as far as I can tell. 

Although it would be complete to say "in bytes" for all other properties, probably not necessary.